From 9976d63113b8c913f736dc70fb91cd23a54c9843 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 12 Mar 2024 12:28:09 +0800
Subject: [PATCH 02/10] FROM ARCHLINUX: fix(VBox/Additions): disable unneeded
 vboxvideo DDX
X-Developer-Signature: v=1; a=openpgp-sha256; l=835; i=xtexchooser@duck.com;
 h=from:subject; bh=J/zsLcRT5hvIROHNnx3olHA79+NIYjRn1TxVCTjb1Qs=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGkvVLY8kwrJn9Ad3yYlb327VemFhauyvOFW/cAtyYr7A
 uz+XJrWUcrCIMbFICumyFJk2ODNqpPOL7qsXBZmDisTyBAGLk4BmMgdF4ZfzB47xI8biN8+lrqT
 1feZnEvAtbtZUVNqTmQKHWix+fKugOGf+u7UD0ZLV8WeWFa6dmWNnoyyw5aly3wfuQpvr2tmfbm
 dFQA=
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

---
 src/VBox/Additions/x11/vboxvideo/Makefile.kmk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/VBox/Additions/x11/vboxvideo/Makefile.kmk b/src/VBox/Additions/x11/vboxvideo/Makefile.kmk
index 82469ea55b42..b3a3d39bc499 100644
--- a/src/VBox/Additions/x11/vboxvideo/Makefile.kmk
+++ b/src/VBox/Additions/x11/vboxvideo/Makefile.kmk
@@ -406,7 +406,7 @@ vboxvideo_drv_118_LIBS += $(vboxvideo_drv_70_LIBS)
 
 ifdef VBOX_USE_SYSTEM_XORG_HEADERS
  # Build using local X.Org headers.  We assume X.Org Server 1.7 or later.
- DLLS    := $(filter-out vboxvideo_drv_%,$(DLLS)) vboxvideo_drv_system
+ DLLS    := $(filter-out vboxvideo_drv_%,$(DLLS))
  SYSMODS := $(filter-out vboxvideo_drv%,$(SYSMODS))
  vboxvideo_drv_system_TEMPLATE = VBoxGuestR3XOrgMod
  vboxvideo_drv_system_CFLAGS := \
-- 
2.46.0

